draft-ietf-masque-connect-udp-listen.md

```diff
@@ -236,7 +236,7 @@ back to the client.
    :scheme = https
    :path = /.well-known/masque/udp/*/*/
    :authority = proxy.example.org
-   connect-udp-listen = 2
+   Connect-UDP-Listen = 2
```

Comment:
In HTTP/2 and HTTP/3, headers are encoded over the wire in lowercase, so I'd revert this one instance